content/docs/guides/flyway-multiple-environments.md

```diff
@@ -0,0 +1,211 @@
+---
+title: Manage multiple database environments
+subtitle: Learn how to create and manage multiple database environments with Neon and Flyway
+enableTableOfContents: true
+---
+
+Flyway is a database migration tool that facilitates version control for databases. It allows developers to manage and track changes to the database schema, ensuring that the database evolves consistently across different environments.
+
+When automating releases, there are often multiple environments or a chain of environments that you must deliver changes to in a particular order. Such environments might include _development_, _staging_, and _production_.
+
+In this guide, we'll show you how to use Neon's branching feature to spin up a branch for each environment and how to configure Flyway to manage schema changes across those environments.
+
+## Prerequisites
+
+- A flyway installation. See [Get started with Flyway and Neon](/doscs/guides/flyway) for installation instructions.
+- A Neon account. See [Sign up](/docs/get-started-with-neon/signing-up).
+- A Neon project. See [Create your first project](/docs/get-started-with-neon/setting-up-a-project).
+- A database. This guide uses the ready-to-use `neondb` database on the `main` branch of your Neon project. You can create your own database if you like. See [Create a database](/docs/manage/databases#create-a-database) for instructions.
+
+## Add a table to your database
+
+Set up a database to work with by adding a table to your `neondb` database on the `main` branch of your Neon project. We'll consider this your _production_ environment database.
+
+Open the [Neon SQL Editor](/docs/get-started-with-neon/query-with-neon-sql-editor), and run the following statement:
+
+```bash
+create table person (
+    ID int not null,
+    NAME varchar(100) not null
+);
+```
+
+## Create databases for development and staging
+
+Using Neon's branching feature, create your _development_ and _staging_ databases. When you create a branch in Neon, you are creating a copy-on-write clone of the parent branch that incudes all databases and roles that exist on the parent, and each branch is an isolated Postgres instance with its own compute resources.
+
+Perform these steps twice, once for your _development_ branch and once for your _staging_ branch.
```

Comment:
Probably good idea to replace CodeTabs for the new Tabs feature here: Console,CLI,API

Reply:
Replaced.